geti_sdk/deployment/deployment.py

```diff
@@ -426,31 +425,12 @@ def generate_ovms_config(self, output_folder: Union[str, os.PathLike]) -> None:
             ovms_model_dir = os.path.join(ovms_models_dir, model_name, model_version)
             source_model_dir = model.model_data_path
 
-            if otx.__version__ >= "1.4.0":
```

Comment:
This should not be removed I believe, it applies for otx versions 1.4 and up. It's function is to embed the preprocessing in the model graph, and save the updated model. 
This is required, otherwise once the model is loaded into OVMS it will not be able to handle the input that we send to it later on.

Reply:
Wow, I probably removed the wrong logical branch, good catch!

Reply:
fixed now